### PR TITLE
Update discovery-context.jsonld

### PR DESCRIPTION
**Pull request by AndreaCimminoArriaga**
Modified the context so it works again, some keywords must be discussed

### CODE REVIEW

**Comment by mmccool** [comment]
So if I understand this correctly, this updates "next" so it is typed as a URI.  Were there any other keyword changes?  Can you summarize what else needs to be discussed?

**Comment by AndreaCimminoArriaga** [comment]
This update fixes the keys `next` and `total`. However, other two keys generate conflicts in the context that are the `@id` and `member`, for the moment I have erased them so the context has no errors and can be used for the round-tripping. I'm not sure if we need to discuss about them (we already agreed on having this alternative way of paginating), they just need to be fixed according to the needs of @wiresio who proposed them and upload the ontology with them.

On the other hand, I would like to discuss today some other terms from our ontology like create and modified since I saw that the new TD spec already includes this terms.
